### ISSUE TITLE
Recompiling of nodes created with TexText < 0.5 fails

### ISSUE THREAD

**Issue by jcwinkler** [textext/textext#15]
When editing nodes created with TexText < 0.5 the following error is thrown:

```
TypeError: unsupported operand type(s) for *=: 'NoneType' and 'float' 
```

Reason: These nodes do not have the scale attribute. Hence, the following code fails, since `current_scale` is `None` (returned by `TexText.get_old()`):

```
current_scale *= self.uutounit(1, "pt")
```

Possible fix: Let `TexText.get_old()` return 1.0 instead of None for the default scale factor.

**Comment by jcwinkler** [comment]
**Hotfix for TexText 0.7.2**

Download link for fixed textext.py to replace installed file from version 0.7.2: [textext.py](https://github.com/textext/textext/blob/releases/0.7/textext.py)

**Comment by jcwinkler** [comment]
Fixed in release 0.7.3
